hbase-server/src/test/java/org/apache/hadoop/hbase/mob/TestExpiredMobFileCleaner.java

```diff
@@ -140,7 +140,8 @@ public void testCleaner() throws Exception {
     assertEquals("Before cleanup without delay 1", 1, firstFiles.length);
     String firstFile = firstFiles[0].getPath().getName();
 
-    ts = EnvironmentEdgeManager.currentTime() - 1 * secondsOfDay() * 1000; // 1 day before
+    ts = (long) (EnvironmentEdgeManager.currentTime() - 1.5 * secondsOfDay() * 1000); // 1.5 day
```

Comment:
Why we need to change the ts here?

Reply:
When the time-to-live (TTL) is set to 2 days (48 hours), the mob data that exists for 1.5 days (36 hours) should be readable. However, in the previous version, only mob data that existed for up to 1 day could be accessed, and mob data that existed for more than 1 day but less than 2 days could not be accessed. Therefore, I have modified the ts setting to 1.5 days.

Reply:
OK, please move the comment to the top of this statement, so it will not be split to two lines.

Reply:
Please fix this minor style issue?

Reply:
Ping @liuwenjing17

Reply:
Done. Sorry for the delay. @Apache9